https://bugs.gentoo.org/542436

From f56ab87b25f2228a67ac592a1c18793c72dd03eb Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 7 Mar 2015 22:29:33 -0500
Subject: [PATCH crda] libreg: link against crypto libs

Since libreg uses funcs from the crypto lib, make sure we link them.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8e345a1..77708e6 100644
--- a/Makefile
+++ b/Makefile
@@ -117,7 +117,7 @@ keys-%.c: utils/key2pub.py $(wildcard $(PUBKEY_DIR)/*.pem)
 
 $(LIBREG): regdb.h reglib.h reglib.c
 	$(NQ) '  CC  ' $@
-	$(Q)$(CC) $(CFLAGS) $(CPPFLAGS) -o $@ -shared -Wl,-soname,$(LIBREG) $^
+	$(Q)$(CC) $(CFLAGS) $(CPPFLAGS) -o $@ -shared -Wl,-soname,$(LIBREG) $^ $(filter-out -lreg,$(LDLIBS))
 
 install-libreg-headers:
 	$(NQ) '  INSTALL  libreg-headers'
-- 
2.3.1

